sdk/translation/Azure.AI.Translation.Document/README.md

```diff
@@ -420,7 +420,7 @@ This project has adopted the [Microsoft Open Source Code of Conduct][code_of_con
 [documenttranslation_nuget_package]: https://www.nuget.org/packages/Azure.AI.Translation.Document
 [documenttranslation_samples]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/translation/Azure.AI.Translation.Document/samples/README.md
 <!-- TODO: Add correct link when available -->
-[documenttranslation_rest_api]: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/TranslatorText/preview/v1.0-preview.1/TranslatorBatch.json
+[documenttranslation_rest_api]: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/TranslatorText/stable/v1.0/TranslatorBatch.json
```

Comment:
Do you include the service version targeted somewhere on your README? we do in python, so just checking.

Reply:
We don't. I looked in the readme and no other reference besides this link. 
Do you think I should add something about it?

Reply:
We have it right by the "Install the Package" info: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/translation/azure-ai-translation-document#install-the-package

Since our README gets published on our MS docs / PyPi page, I think it's nice to have so you can quickly map SDK version -> service version. But if you don't normally do that in .NET then ignore me!

Reply:
Added